server/http/restapi/marketplace_test.go

```diff
@@ -0,0 +1,90 @@
+package restapi
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMergeApps(t *testing.T) {
+	for name, test := range map[string]struct {
+		a        []MarketplaceApp
+		b        []MarketplaceApp
+		expected []MarketplaceApp
+	}{
+		"same slices": {
```

Comment:
Just for completion, maybe add a test with both slices empty?

Reply:
:+1: added in a429ba6